README.md

```diff
@@ -543,7 +543,7 @@ $ grep Version: kernel.spec
 Version:        5.4.91
 
 # Download the associated tar.gz file.  Be sure to substitute your version number in the URL here
-$ wget  https://github.com/microsoft/WSL2-Linux-Kernel/archive/linux-msft-5.4.91.tar.gz
+$ wget https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/rolling-lts/mariner/5.4.91.tar.gz

```

Comment:
I did not realize at first that this change will make it so that the file downloaded now will be saved as `5.4.91.tar.gz`, but the kernel spec file expects it to be in format of `{name}-version.tar.gz`. I would suggest adding command to retain the name.
```suggestion
$ wget -O linux-msft-5.4.91.tar.gz https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/rolling-lts/mariner/5.4.91.tar.gz
```